### ISSUE TITLE
Question: Is this intended to replace debugbar?

### ISSUE THREAD

**Issue by dambridge** [fruitcake/laravel-telescope-toolbar#26]

**Comment by barryvdh** [comment]
Answer: no

**Comment by adevade** [comment]
@barryvdh If you do have a minute over, could you explain the differences between the packages?
Maybe an idea to have a section about it in the readme?

**Comment by barryvdh** [comment]
Yeah I should probably do that. It's mainly lighter than Debugbar, with regard of load time and amount of external libraries (jquery etc). Telescope Toolbar uses the data gathered by Telescope itself.

**Comment by ziming** [comment]
If I have Telescope installed, does that mean I cannot use laravel debug bar and must use this? Is telescope toolbar better than debugbar if i have telescope installed?

**Comment by barryvdh** [comment]
No you can use it. But using Telescope and Debugbar will just have a bit more overhead.

**Comment by ziming** [comment]
Which give me more features overall?

Telescope + Debugbar

or

Telescope + Telescope toolbar?

**Comment by barryvdh** [comment]
Telescope Toolbar is just a front-end for the Telescope data. It does not add any extra features to Telescope itself. Debugbar is just a bit different, it's easier to add custom collectors (model count, livewire etc).

**Comment by Braunson** [comment]
@ziming A great post written by @barryvdh comparing each other which should resolve any questions you have.

https://barryvdh.nl/laravel/debugbar/telescope/2020/06/14/laravel-debugbar-vs-telescope-toolbar/
